instrumentation/jdbi3/pom.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0"?>
+<!--
+
+    Copyright The OpenZipkin Authors
+    SPDX-License-Identifier: Apache-2.0
+
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <parent>
+    <groupId>io.zipkin.brave</groupId>
+    <artifactId>brave-instrumentation-parent</artifactId>
+    <version>6.2.0-SNAPSHOT</version>
```

Comment:
@reftel sincere apologies, the `6.2.1-SNAPSHOT` is the current version (6.2.0 already sailed), what @codefromthecrypt meant is to bump all POMs to `6.3.0-SNAPSHOT` since a new module being added. Thank you and apologies for some confusion here

Reply:
No problem! =) I´ll include bumping all versions to `6.3.0-SNAPSHOT` that in the next push.

Reply:
cool. I'll be offline for a long weekend so if you manage before singapore time evening I can help merge. otherwise maybe @reta who is later in the globe

Reply:
No stress! =) Enjoy your time off!